### ISSUE TITLE
[TSDoc> Playground]: Incorrect method of navigation is present for the TAB items present under Playground.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#249]
**Test Environment:**
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/play/

**Pre-Requisite:**
Navigate through keyboard only.

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Press the tab key and navigate to the 'PlayGround' option and press enter key to activate it.
3. Press the tab key and navigate to the tab items present at right hand side and observe the issue.

**Actual Result:**
Incorrect method of navigation is present for the TAB items present under 'Playgrounf' as on pressing tab focus is moving to next tab item.

**Expected Result:**
Proper and correct method of navigation should be present for the tab items present under 'Playground' all the tab items should be navigable with Left/Right arrow key.

**User Experience:**
Inefficient or incorrect navigation methods for files and folders within a table can lead to user confusion and frustration, hindering their ability to locate and interact with specific items effectively.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-order
 
Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment.

![Incorrect method of navigation is present for the TAB items present under Playground](https://github.com/microsoft/rushstack-websites/assets/95913748/033ac180-982d-4f0e-9f91-6edf3ed9c061)
